README.md

```diff
@@ -805,19 +805,23 @@ class User extends Model
     }
 }
 ```
+**Warning:** naming the foreign key same as the relation name will prevent the relation for being called on dynamic property, i.e. in the example above if you replaced `group_ids` with `groups` calling `$user->groups` will return the column instead of the relation.
```

Comment:
I think "e.g." would be more appropriate because you're providing one possible example.

See: https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference